config/default.js

```diff
@@ -375,7 +375,7 @@ module.exports = {
   // The withExperiment HOC relies on this config to enable/disable A/B
   // experiments.
   experiments: {
-    installButtonWarning: false,
+    installButtonWarning2: false,
```

Comment:
I'd suggest a different (more meaningful) name like `installWarningWith4Branches` but up to you.

Reply:
It's the second iteration, and we could have more later, which might also have 4 branches, so I think a simple `2` is probably the most meaningful way to label it, but thanks for the suggestion. I'm also thinking that perhaps in the future we will keep the `id`s in `default.js`, with a value of `false`, which will make it easier to see which `id`s have been used previously.

Reply:
Actually, I think I'll do that right now in fact. I'll update the patch to add `installButtonWarning2` in addition to `installButtonWarning`, rather than replace `installButtonWarning`.

Reply:
mmm ok, then I suggest you link to the issue above it then.

Reply:
Which issue? The current one, that is about making `id`s unique, or the issue that implemented this new version of the experiment? If we're going to keep adding new `id`s are you proposing that we'd add a link to the issue for each one as well?

Reply:
The issue related to the new experiment.

> If we're going to keep adding new ids are you proposing that we'd add a link to the issue for each one as well?

Yes, so that we document the "why" (we need a new id that looks similar to exist ones).

Reply:
Fair enough. I'll add the comment.